backend/packages/worker/src/worker/parsing/storage.py

```
@@ -0,0 +1,35 @@
+        return key
+
+    def get_artifact(self, doc_id: str) -> Optional[ParsedArtifact]:
+        key = self._get_key(doc_id)
+        try:
+            response = self.s3_client.get_object(Bucket=self.bucket_name, Key=key)
+            payload = response['Body'].read().decode('utf-8')
+            return ParsedArtifact.model_validate_json(payload)
+        except ClientError as e:
+            if e.response['Error']['Code'] == 'NoSuchKey':
+                return None
+            raise e
```

Ruff (0.15.11)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)